Makefile

```diff
@@ -84,6 +84,7 @@ $(call set-feature, Zifencei)
 
 # Experimental SDL oriented system calls
 ENABLE_SDL ?= 1
+ifneq ("$(CC_IS_EMCC)", "1")
```

Comment:
Why does the detection of SDL headers/library shipped with Emscripten rely on sdl2-config?

Reply:
Got it. Add some comments in `Makefile` to address the relationship between system SDL headers and the emscripten counterpart.